packages/11ty/_includes/components/icons-cc/index.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs')
+const path = require('path')
+const ccIcons = fs
+  .readdirSync(path.join(__dirname, 'icons'))
+  .map((filename) => {
+    const id = path.basename(filename, '.svg')
+    return fs.readFileSync(
+      path.join(__dirname, 'icons', filename),
+      { encoding: 'utf8' }
+    ).replace('<svg', `<symbol id="${id}"`).replace('</svg', '</symbol')
```

Comment:
this double replace to change both opening and closing tags from `svg` to `symbol` is a little eh